api/v1alpha1/clusterextension_types.go

```diff
@@ -125,6 +125,7 @@ const (
 	// TODO(user): add more Types, here and into init()
 	TypeInstalled = "Installed"
 	TypeResolved  = "Resolved"
+	TypeManaged   = "Managed"
```

Comment:
I don't love that we're adding a new condition type for this. It feels too implementation-specific and not really something that an end user necessarily needs to know details of.

Maybe a better option:
- Change the name to `Healthy` (reasoning: we're going to re-add support for health checks in the future. If we don't have watches established, we will say "Healthy: Unknown". And if we have such strong coupling between two different condition types (i.e. `Managed=False` causes `Healthy: Unknown`), then we don't need both.

Reply:
One point against this idea could be: "what do we say if we _did_ successfully establish watches".

I think there are two options:
1. Make sure the `Healthy` condition is deleted from the list (and yeah, I know this violates our "set every condition, every time" rule. We'd have to square that.)
2. Set `Healthy: Unknown` with reason: "health checks are not yet implemented" (I don't love this).

Reply:
As discussed in sync, make this condition `Healthy`. It is not present when watches are successfully established. It is set to `Unknown` when the watches are not able to be established and the error message is populated.

Reply:
updated the condition logic to follow this. Also updated our e2e tests to no longer enforce the expectation that all status conditions are always present